library.sh

```diff
@@ -632,6 +634,9 @@ function go_update_deps() {
     fi
   fi
 
+  group "Set Go version"
+  go mod edit -go=${GOVERSION}
```

Comment:
I guess other repos dont have issues with moving to 1.17. So better do it for all.